src/Nethermind/Nethermind.DataMarketplace.Infrastructure/NdmRequiredServices.cs

```diff
@@ -15,6 +15,7 @@
 //  along with the Nethermind. If not, see <http://www.gnu.org/licenses/>.
 
 using Nethermind.Blockchain;
+using Nethermind.Blockchain.Bloom;
```

Comment:
do not toucm NDM please - there will be lots of conflicts

Reply:
Had to, doing a minimum.